src/OpenTK/NativeWindow.cs

```diff
@@ -165,18 +166,9 @@ public Point PointToScreen(Point point)
         /// <summary>
         /// Processes operating system events until the NativeWindow becomes idle.
         /// </summary>
-        public void ProcessEvents()
+        public void ProcessEvents ()
```

Comment:
Nicer if you can not make whitespace changes when changing other things, keeps git history cleaner.